config/routes.rb

```diff
@@ -2527,6 +2527,7 @@
         show
         show_list
         stacks_ot_info
+        stacks_ot_copy
```

Comment:
As we are not using http.post, We still need this?

Reply:
Submitting the forms seems to not work after removing this line.